### PR TITLE
Feat/delete screenshots

### PR DESCRIPTION
**Pull request by adamtretera**

### CODE REVIEW

File: src/modules/auth/HOC/firewall.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { React, useSelector, Auth } from '../dependencies';
-
+import Loading from '../components/Loading';
```
Comment (baierjak) [inline src/modules/auth/HOC/firewall.jsx:2]: This component should be part of UI module

File: src/modules/ui/components/CardDetail/CardDetail.jsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { FormattedMessage, FormattedDate } from 'react-intl';
 import * as felaRules from './CardDetail.styles';
 import Button from '../Button';
 import NoteForm from '../NoteForm';
+import DeleteButton from '../../../reports/components/DeleteButton';
```
Comment (baierjak) [inline src/modules/ui/components/CardDetail/CardDetail.jsx:11]: This is imported from different module so it should be like this
```suggestion
import { DeleteButton } from 'modules/reports';
```

Comment (baierjak) [reply src/modules/ui/components/CardDetail/CardDetail.jsx:11]: be sure DeleteButton is exported from that module via its index.js

File: src/modules/ui/components/CardDetail/CardDetail.jsx [hunk 3]
```diff
@@ -38,12 +40,11 @@ const CardDetail = ({ message, onCancel, visible }) => {
...
                 footer={[
                     <div key={'id'}>
-                        <Button type="secondary">
-                            <FormattedMessage id="card.button.delete" />
-                        </Button>
+                        <DeleteButton onCancel={onCancel} id={id} />
                         <Button type="primary" icon={<CopyOutlined />}>
                             <FormattedMessage id="card.button.copy" />
                         </Button>
+                        <Button>sdasd</Button>
```
Comment (baierjak) [inline src/modules/ui/components/CardDetail/CardDetail.jsx:47]: eh?

File: package.json [hunk 4]
```diff
@@ -40,6 +40,7 @@
         "react-redux": "^7.2.2",
         "react-router-dom": "^5.2.0",
         "react-scripts": "4.0.3",
+        "react-uuid": "^1.0.2",
```
Comment (baierjak) [inline package.json:43]: See this - https://blog.bitsrc.io/why-is-nanoid-replacing-uuid-1b5100e62ed2

File: src/modules/auth/HOC/firewall.jsx (hunk 1 above)
Comment (adamtretera) [reply src/modules/auth/HOC/firewall.jsx:2]: Working on it in #10

File: src/modules/ui/components/CardDetail/CardDetail.jsx (hunk 3 above)
Comment (adamtretera) [reply src/modules/ui/components/CardDetail/CardDetail.jsx:47]: Removed

File: package.json (hunk 4 above)
Comment (adamtretera) [reply package.json:43]: Agree - changed

File: src/modules/auth/HOC/firewall.jsx (hunk 1 above)
Comment (baierjak) [reply src/modules/auth/HOC/firewall.jsx:2]: cool

**Pull request merged**